packages/@coorpacademy-components/src/molecule/drag-and-drop/common.css

```diff
@@ -9,7 +9,7 @@
 }
 
 .box {
-  width: 360px;
+  width: 436px;
```

Comment:
imho every component should be flex, and the parent is responsible to set the width (eventually depending on media queries)